### ISSUE TITLE
Navigate images within "Crocoblock Gallery Grid Widget" Lightbox

### ISSUE THREAD

**Issue by enuzzo** [CrocoBlock/suggestions#3698]
In "Crocoblock Gallery Grid Widget" if you enable the "Gallery" ticker, you'll be able to open a lightbox and if you click on the search icon, you'll get a scrollable lightbox gallery.

It would be useful to get the scrollable gallery clicking on the whole image, not only the icon.

Since we already have the ability to disable "Gallery" and click every single image without browsing, it would be super useful to just click the whole picture to have a scrollable lightbox gallery.

As a good UX/UI advice, people browsing would expect to click on the image itself to open a bigger view and scroll to other images in the gallery. It won't be easily understood that clicking the image will not scroll and the "search icon" does otherwise.

**Comment by enuzzo** [comment]
I noticed another issue: while in the Elementor Lightbox settings you can also increase the control icon sizes and colors, in Crocoblock you still can't. This way, if you set your PhotoSwipe background to be light or white, the controls won't change their colour, therefore remain mostly invisible.

TL;DR
- Ability to change size and colour of controls of PhotoSwipe 
- Nice to have: a trigger to "Use Elementor's default Lightbox styles" that would benefit consistency among the website
